asset/sass/_elements.scss

```diff
@@ -30,3 +30,18 @@ pre[class*="language-"] {
     margin-top: $padding-large-vertical * 2;
     margin-bottom: $padding-large-vertical * 2;
 }
+
+.dropdown_packagename {
```

Comment:
This additions should be moved under the `blocks` folder. (a new file like `_choices.scss` or added to the file `_component-selector.scss`)

Reply:
And the name should follow the block: `choices__packagename

Reply:
Done.